android/src/toga_android/widgets/base.py

```diff
@@ -126,21 +125,19 @@ def set_font(self, font):
         # By default, font can't be changed
         pass
 
-    # Although setBackgroundColor is defined in the View base class, we can't use it as
-    # a default implementation because it often overwrites other aspects of the widget's
-    # appearance. So each widget must decide how to implement this method, possibly
-    # using one of the utility functions below.
     def set_background_color(self, color):
-        self.set_background_simple(color)
-
-    def set_background_simple(self, color):
+        # Set background to None, when TRANSPARENT is requested, in order to prevent
+        # clipping of ripple and other effects on widgets.
         self.native_toplevel.setBackground(
-            ColorDrawable(
-                self._default_background_color if color is None else native_color(color)
-            )
+            None if color in (None, TRANSPARENT) else ColorDrawable(native_color(color))
         )
 
     def set_background_filter(self, color):
+        # Although setBackgroundColor is defined in the View base class, we can't
+        # use it as a default implementation on some widgets(e.g. Button), because
+        # it often overwrites other aspects of the widget's appearance. For example,
+        # when setBackgroundColor is used on a Button, it changes the background behind
+        # the button rather than the button's actual color.
```

Comment:
```suggestion
        # use it as a default implementation on some widgets (e.g. Button), because
        # it often overwrites other aspects of the widget's appearance. For example,
        # when setBackgroundColor is used on a Button, it makes the button appear
        # as a solid rectangle with no bevels or animations.
```